recipes/pescador/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = "1.0.0" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "5f3663fe66ce18f3f06be9ba5cbfbe1a" %}
```

Comment:
Please update this to be sha256 instead of md5. [conda-forge prefers sha256](https://github.com/conda-forge/staged-recipes/blob/master/README.md#2-how-do-i-populate-the-hash-field)